File: VB/dxSampleT590924/Models/Order.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace dxSampleT590924.Models
	Public Class Order
		Public Property OrderID() As Integer
		Public Property OrderDate() As Date
		Public Property CustomerID() As String
		Public Property CustomerName() As String
		Public Property ShipCountry() As String
		Public Property ShipCity() As String
	End Class
	Public Class Customer
		<Required>
		Public Property CustomerID() As Integer

		<Required, MaxLength(10)>
		Public Property FirstName() As String
		<Required, MaxLength(10)>
		Public Property LastName() As String
		Public Property HireDate() As Date
		Public Property Orders() As List(Of Order)
	End Class
End Namespace

File: CS/dxSampleT590924/Models/Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace dxSampleT590924.Models {
    public class Order
    {
        public int OrderID { get; set; }
        public DateTime OrderDate { get; set; }
        public string CustomerID { get; set; }
        public string CustomerName { get; set; }
        public string ShipCountry { get; set; }
        public string ShipCity { get; set; }
    }
    public class Customer
    {
        [Required]
        public int CustomerID { get; set; }

        [Required]
        [MaxLength(10)]
        public string FirstName { get; set; }
        [Required]
        [MaxLength(10)]
        public string LastName { get; set; }
        public DateTime HireDate { get; set; }
        public List<Order> Orders { get; set; }
    }
}
